frontend/src/component/pages/Home.tsx

```diff
@@ -267,10 +285,17 @@ export class Home extends Component<Props, State> {
           </Section>
         </div>
         <Footer
+          uiFactory={this.props.uiFactory}
+          handleShowChangeLogModal={this.handleShowChangeLogModal}
```

Comment:
```suggestion
          onShowChangeLogBtnClick={this.handleShowChangeLogBtnClick}
```

Reply:
This leaves the flexibility for us to replace the modal with something else without changing the rest of the code.